### PR TITLE
78 display tab text

### PR DESCRIPTION
**Pull request by ngaruko**
# Description

[description]

https://github.com/medic/medic-release-testing/issues/78

# Code review checklist

- [ ] Readable: Concise, well named, follows the [style guide](https://docs.communityhealthtoolkit.org/contribute/code/style-guide/), documented if necessary.
- [ ] Documented: Configuration and user documentation on [cht-docs](https://github.com/medic/cht-docs/)
- [ ] Tested: Unit and/or e2e where appropriate
- [ ] Internationalised: All user facing text
- [ ] Backwards compatible: Works with existing data and configuration or includes a migration. Any breaking changes documented in the release notes.

# License

The software is provided under AGPL-3.0. Contributions to this project are accepted under the same license.

### CODE REVIEW

**Review by dianabarsan — CHANGES_REQUESTED** [review]
Nice work! I left a couple of comments inline.

File: tests/mobile/common/common.specs.js [hunk 1]
```diff
@@ -63,4 +64,70 @@ describe('Navigation tests : ', () => {
...
+      jasmine.DEFAULT_TIMEOUT_INTERVAL = originalTimeout;
+    });
+
+    beforeAll(async () => {
+      await utils.saveDoc(district);
+      await utils.createUsers([offlineUser]);
+    });
+
+    it('No tab text labels displayed  on mobile view for over 3 tabs', async () => {
+      await browser.driver.manage().window().setSize(389, 500);
```
[the remark this answers is not in the document]
Comment (ngaruko) [reply tests/mobile/common/common.specs.js]: done

**Review by dianabarsan — COMMENTED** [review]
I think I spotted something that might fix your build. Also, please rebase off of master for the driver version fix.

File: tests/mobile/common/common.specs.js [hunk 2]
```diff
@@ -63,4 +65,71 @@ describe('Navigation tests : ', () => {
...
+      expect(tabTexts.length).toBe(5);
+      expect(tabTexts).toEqual([ '', '', '', '', '' ]);
+    });
+
+    it('Display page tab text labels even on mobile view, whenever there are 3 or fewer tabs', async () => {
+      //change permissions
+      await utils.updateSettings({permissions:newPermissions});
+      await commonElements.goToLoginPageNative();
+      await loginPage.loginNative(user.username, user.password);
+      await commonElements.waitForLoaderToDisappear(200000);
```
Comment (dianabarsan) [inline tests/mobile/common/common.specs.js:127]: I think this is an extreme timeout. It also looks like all tests that follow this one fail with an error about not being able to load settings. 
I think you need to do this in the afterAll of this test: 
1. Delete the user (with utis.deleteUsers)
2. Logout and login as admin

**Review by dianabarsan — CHANGES_REQUESTED** [review]
One idea in line about the failing e2e tests.

**Review by dianabarsan — APPROVED** [review]
Awesome work!!

**Pull request merged**